### PR TITLE
Add contents of mid-December install and survey email

### PR DESCRIPTION
**Pull request by stargaser**
The contents of this email will be sent by bcc: to registered workshop participants on Dec 18 or 19.

* Pre-workshop survey
* Installation instructions
* Contacting us with problems or questions

### CODE REVIEW

**Comment by stargaser** [comment]
Is this something to ask @lglattly to copy-edit?

File: 00-Install_and_Setup/00_INSTALL_EMAIL.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+__Pre-workshop Survey__
+
+Please fill out our survey at https://forms.gle/FUA1Bx17WgDjCKk27 regarding your past experience with astronomical Python as we can better ensure the workshop is tailored to your needs.
+
+__Setup Instructions__
+https://github.com/astropy/astropy-workshop/blob/master/00-Install_and_Setup
+
+Please be sure your laptop is properly configured BEFORE the workshop. The setup could take as long as one hour depending on your current configuration and internet speeds. If you have any trouble, please open an issue in the repo using https://github.com/astropy/astropy-workshop#problems-or-questions or post a message on the #workshop Slack channel in the Astropy workspace (described below). If on the morning of the workshop, you still have doubts or questions about your laptop setup, please arrive promptly at 9am.
+
+Our facilitators are still working the content of the workshop notebooks. We are pleased to be able to present features from our new 4.0 release of the astropy core package. We will send one more email just before the workshop with instructions to update to the final version of the notebooks.
```
Comment (pllim) [inline 00-Install_and_Setup/00_INSTALL_EMAIL.md]:
```suggestion
Our facilitators are still on working the content of the workshop notebooks. We are pleased to be able to present features from our new 4.0 release of the astropy core package. We will send one more email just before the workshop with instructions to update to the final version of the notebooks.
```

File: 00-Install_and_Setup/00_INSTALL_EMAIL.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+This workshop is sold out and we are anticipating folks to show up who did not pre-register. If you do not arrive by 10:30, your seat may be given away. If you know you're going to be late, please drop me an email.
+
+Continental breakfast will be available from 9-10:30 and there will be snacks and coffee again in the afternoon. Lunch will NOT be provided.
+
+Slack Channel
+http://astropy-slack-invite.herokuapp.com/
+Please join the #workshop channel in the Astropy slack. 
+
+Looking forward!
```
Comment (pllim) [inline 00-Install_and_Setup/00_INSTALL_EMAIL.md]:
```suggestion
Looking forward to seeing you there!
```

**Comment by pllim** [comment]
I see that the email is sent out. Should we merge this then?

**Comment by stargaser** [comment]
Uh, yes, would you please merge it @pllim?

**Pull request merged**